README.md

```diff
@@ -103,3 +146,12 @@ A fuzzing engine launcher script receives configuration through the following en
 | `FUZZER_OUTPUT_ROOT`       | A writable path that can be used by the fuzzer during its execution (e.g., as a workspace or for generated artifacts). See the variables below for specific categories of output. |
 | `FUZZER_OUTPUT_CORPUS_DIR` | A path under `FUZZER_OUTPUT_ROOT` where the new generated tests should be stored. |
 | `FUZZER_ARTIFACTS_DIR`     | A path under `FUZZER_OUTPUT_ROOT` where generated crashes and other relevant artifacts should be stored. |
+
+## Rule reference
+
+* [`cc_fuzz_test`](/docs/cc-fuzzing-rules.md#cc_fuzz_test)
+* [`cc_fuzzing_engine`](/docs/cc-fuzzing-rules.md#cc_fuzzing_engine)
+
+## Disclaimer
+
+This is not an officially supported Google product.
```

Comment:
Do we want this disclaimer, this will be supported, and is an important part of our fuzzing portolio ?

Reply:
As long as we have someone maintaining it, fixing bugs, then it is not needed. I havent seen it used often, e.g. https://github.com/dvyukov/go-fuzz
Upto you. Adding it decreases credibility slightly as well.

Reply:
Good point. I'm also seeing now that other Bazel rules repositories don't mention that either, so I'm removing it from here as well.